docs/model_serving/model_serving.rst

```diff
@@ -0,0 +1,419 @@
+
+.. image:: routings-count-changed.png
+   :align: center
+   :alt: Edit model service dialog
+
+Terminating Model Service
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The model service periodically runs a scheduler to adjust the routing
+count to match the desired session count. However, this puts a burden on
+the Backend.AI scheduler. Therefore, it is recommended to terminate the 
+model service if it is no longer needed. To terminate the model service, 
+click on the trash icon in the Control tab. A modal will appear asking 
+for confirmation to terminate the model service. Clicking ``OK`` 
+will terminate the model service. The terminated model service will be 
+removed from the list of model services.
+
+.. image:: terminate-model-service-dialog.png
+   :width: 500
+   :align: center
+   :alt: Terminate model service dialog
+
+Handling Failed Model Service Creation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If the status of the model service remains ``UNHEALTHY``, it indicates
+that the model service cannot be executed properly.
+
+The common reasons for creation failure and their solutions are as
+follows:
+
+-  Insufficient allocated resources for the routing when creating the
+   model service
+
+   -  Solution: Terminate the problematic service and recreate it with
+      an allocation of more sufficient resources than the previous
+      settings.
+
+-  Incorrect format of the model definition file (``model-definition.yml``)
+
+   -  Solution: Verify the format of the model definition file (link)
+      and if any key-value pairs are incorrect, modify them and
+      overwrite the file in the saved location. Then, click the refresh
+      button to ensure that the routing of the model service is set
+      correctly.
+
+      .. image:: refresh-button.png
+         :align: center
+         :alt: Refresh button
+
+Generating Tokens
+~~~~~~~~~~~~~~~~~
+
+Once the model service is successfully executed, the status will be set
+to ``HEALTHY``. In this case, you can click on the corresponding endpoint
+name in the Model Service tab to view detailed information about the
+model service. From there, you can check the service endpoint in the
+routing information of the model service. If the “Open to Public” option
+is enabled when the service is created, the endpoint will be publicly
+accessible without any separate token, and end users can access it.
+However, if it is disabled, you can issue a token as described below to
+verify that the service is running properly.
+
+.. image:: routing-page.png
+   :align: center
+   :alt: Routing page
+
+Click the token creation button located to the right of the generated
+token list in the routing information. In the modal that appears for
+token creation, enter the expiration date. The issued token will be
+added to the list of generated tokens. Click the copy icon in the token
+item to copy the token, and add it as the value of the following key.
+
+.. image:: edit-model-service-dialog.png
```

Comment:
This image is not related to `Generating Token`.

Reply:
<img width="806" alt="image" src="https://github.com/lablup/backend.ai-docs-webui/assets/621215/0aac0763-a34b-4d94-b6c9-8c4ad7e6427a">